### PR TITLE
fix(ui): added effect to topnav button

### LINKED ISSUE #2259: [Feature] (UI): Request to add micro-interaction to top nav-bar buttons for better User experience
### Feature description

Currently, the top nav-bar buttons on Alby has no interaction when users hover on them. This doesn’t seem to be the best user experience for users of Alby.

https://user-images.githubusercontent.com/105345830/227198703-e8814146-120c-4db4-9e04-13ca1399aba7.mov

### Describe the solution

Would love to add a micro-interaction to the buttons when users hover on a particular button.

Like this: 

https://user-images.githubusercontent.com/105345830/227202986-1747c2da-c0f8-461f-b31f-1cfd8c096b69.mov

### Describe alternatives

_No response_

### Additional context

_No response_

### Are you working on this?

Yes

### PR DESCRIPTION
**Pull request by HARI-124**
### Describe the changes you have made in this PR

I have added hover effect for the navbar elements to improve the user experience as mentioned in the issue #2259 

### Link this PR to an issue [optional]

Fixes #2259

### Type of change

(Remove other not matching type)

- `fix`: Bug fix (non-breaking change which fixes an issue)

### Screenshots of the changes [optional]

https://user-images.githubusercontent.com/97289088/230618233-95f65638-feac-419d-90a2-9490e86ad1d4.mp4

### How has this been tested?

I have checked from the browser and I have attached a video regarding that.

### Checklist

- [ ] My code follows the style guidelines of this project and performed a self-review of my own code
- [ ] New and existing tests pass locally with my changes
- [ ] I checked if I need to make corresponding changes to the documentation (and made those changes if needed)

### CODE REVIEW

File: src/app/components/Navbar/NavbarLink.tsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ function NavbarLink({ children, end = false, href }: Props) {
       to={href}
       className={({ isActive }) =>
         classNames(
-          "block font-semibold px-1 text-md",
+          "block font-semibold hover:text-slate-900 transition-all  px-1 text-md",
```
Comment (reneaaron) [inline src/app/components/Navbar/NavbarLink.tsx:17]: Can you please use the colors defined in #2259? 

> Let's use gray600 for light theme and gray300 for dark for hover states, so it's not the same as active. I already updated Master file with this

**Review by reneaaron — CHANGES_REQUESTED** [review]

File: src/app/components/Navbar/NavbarLink.tsx (hunk 1 above)
Comment (HARI-124) [reply src/app/components/Navbar/NavbarLink.tsx:17]: sure I'll update

**Review by reneaaron — APPROVED** [review]
Added a minor fix for the transition class. 

tACK

**Comment by reneaaron** [comment]
Thanks for your contribution @HARI-124! 🐝

**Pull request merged**